_rules/target-size-enhanced-gi8qkf.md

```diff
@@ -0,0 +1,731 @@
+---
+id: gi8qkf
+name:
+rule_type: atomic
+description: |
+  This rule checks that elements that can receive pointer events have a size of at least 44×44 pixels.
+accessibility_requirements:
+  wcag21:2.5.5: # Target size (enhanced) (AAA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag22:2.5.8: # Target Size (Minimum)
+    secondary: true
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+    - Wilco Fiers
+  test_assets: Map Image by <a href="https://www.freepik.com/free-vector/black-white-town-navigation-map_5663353.htm#query=map%20background&position=27&from_view=keyword&track=ais">Freepik</a>
+---
+
+## Applicability
+
+This rule applies to any [HTML element][namespaced element] which [can be targeted by a pointer event][].
+
+Exception: the target is a [User Agent controlled component][] (WCAG exception).
+Exception: not for `area` element (due to weird shapes + often essential size).
+Exception: not if a descendant is focusable (hard to define the clickable area).
+Exception: the target is a [shape-shifted element][] (weird shape).
+
+## Expectation
+
+For each test target, at least one of the following is true:
+
+- the target element has a [clickable area][] containing a rectangle aligned with the screen border whose width and height are at least 44 CSS pixels; or
```

Comment:
"Aligned with the screen border" feels off to me. Screens aren't necessarily rectangles. Some smart watches have circular screens for example. But this also seems like it would require some new input aspect of having to know the something about the edge of the screen?